### PR TITLE
Docs - Upgrade version and release note

### PR DESCRIPTION
**Pull request by yukirora**
**Description**

Upgrade version and release note.

**Major Revision**
- Upgrade package versions
- Add release note for v0.11.0

### CODE REVIEW

File: website/package-lock.json [hunk 1]
```diff
@@ -7467,8 +7467,8 @@
       }
     },
     "node-forge": {
-      "version": "0.10.0",
-      "resolved": "https://registry.npmjs.org/node-forge/-/node-forge-0.10.0.tgz",
+      "version": "0.11.0",
+      "resolved": "https://registry.npmjs.org/node-forge/-/node-forge-0.11.0.tgz",
```
Comment (abuccts) [inline website/package-lock.json:7471]: pls revert

File: website/package-lock.json [hunk 2]
```diff
@@ -9428,7 +9428,7 @@
       "resolved": "https://registry.npmjs.org/selfsigned/-/selfsigned-1.10.11.tgz",
       "integrity": "sha512-aVmbPOfViZqOZPgRBT0+3u4yZFHpmnIghLMlAcb5/xhp5ZtB/RVnKhz5vl2M32CLXAqR4kha9zfhNg0Lf/sxKA==",
       "requires": {
-        "node-forge": "^0.10.0"
+        "node-forge": "^0.11.0"
```
Comment (abuccts) [inline website/package-lock.json:9431]: same

File: superbench/config/azure_ndmv4.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # Azure NDm A100 v4
```
Comment (abuccts) [inline superbench/config/azure_ndmv4.yaml:3]: Can we add more examples on h100 and mi200 skus?

File: docs/user-tutorial/container-images.mdx [hunk 4]
```diff
@@ -50,7 +50,7 @@ available tags are listed below for all stable versions.
 
 | Tag                           | Description                                      |
 |-------------------------------|--------------------------------------------------|
-| v0.10.0-rocm5.7               | SuperBench v0.10.0 with ROCm 5.7                 |
+| v0.11.0-rocm5.7               | SuperBench v0.11.0 with ROCm 5.7                 |
 | v0.9.0-rocm5.1.3              | SuperBench v0.9.0 with ROCm 5.1.3                |
```
Comment (abuccts) [inline docs/user-tutorial/container-images.mdx:54]: add new version but don't remove the old one?

File: docs/user-tutorial/container-images.mdx [hunk 5]
```diff
@@ -30,8 +30,8 @@ available tags are listed below for all stable versions.
 
 | Tag                | Description                         |
 |--------------------|-------------------------------------|
-| v0.10.0-cuda12.2   | SuperBench v0.10.0 with CUDA 12.2   |
-| v0.10.0-cuda11.1.1 | SuperBench v0.10.0 with CUDA 11.1.1 |
+| v0.11.0-cuda12.2   | SuperBench v0.11.0 with CUDA 12.2   |
+| v0.11.0-cuda11.1.1 | SuperBench v0.11.0 with CUDA 11.1.1 |
```
Comment (abuccts) [inline docs/user-tutorial/container-images.mdx:34]: same

File: superbench/config/azure_ndmv4.yaml (hunk 3 above)
Comment (yukirora) [reply superbench/config/azure_ndmv4.yaml:3]: added example file for NDV5 and mi300x to include hipblast/cublaslt/fp8 training/dist-inference

File: superbench/config/azure_ndv5.yaml [hunk 6]
```diff
@@ -0,0 +1,305 @@
+# SuperBench Config
```
Comment (cp5555) [inline superbench/config/azure_ndv5.yaml:1]: It's like a separated PR for NDv5 and MI300?

File: website/blog/2024-09-20-release-0-11.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+tags: [superbench, announcement, release]
+---
+
+We are very happy to announce that **SuperBench 0.11.0 version** is officially released today!
+
+You can install and try superbench by following [Getting Started Tutorial](https://microsoft.github.io/superbenchmark/docs/getting-started/installation).
+
+## SuperBench 0.11.0 Release Notes
+
+### SuperBench Improvements
```
Comment (cp5555) [inline website/blog/2024-09-20-release-0-11.md:17]: I think we don't inlcude all?

File: superbench/config/azure_ndv5.yaml (hunk 6 above)
Comment (yukirora) [reply superbench/config/azure_ndv5.yaml]: seperated into a new PR

File: website/blog/2024-09-20-release-0-11.md (hunk 7 above)
Comment (yukirora) [reply website/blog/2024-09-20-release-0-11.md:17]: updated

**Review by cp5555 — APPROVED** [review]

File: docs/getting-started/run-superbench.md [hunk 8]
```diff
@@ -27,7 +27,7 @@ sb deploy -f remote.ini --host-password [password]
 :::note Note
 You should deploy corresponding Docker image to use release version, for example,
 
-`sb deploy -f local.ini -i superbench/superbench:v0.10.0-cuda12.2`
+`sb deploy -f local.ini -i superbench/superbench:v0.11.0-cuda12.2`
```
Comment (abuccts) [inline docs/getting-started/run-superbench.md:30]: update to cuda12.4?

**Review by abuccts — APPROVED** [review]

File: docs/getting-started/run-superbench.md (hunk 8 above)
Comment (yukirora) [reply docs/getting-started/run-superbench.md]: updated, thanks

**Pull request merged**